### PR TITLE
add details for automl installation

### PR DESCRIPTION
**Pull request by yushan111**
main changes
1. add readme for example
2. add installation details in automl doc (pyzoo/zoo/automl/README)
3. add  automl dependencies in setup. User can use `pip install analytics_zoo.whl[automl]` to install dependencies.

### CODE REVIEW

File: apps/automl/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Jupyter Notebook
+* Matplotlib
+
+## Install Analytics Zoo AutoML
+Analytics Zoo AutoML is still in experimental stage. And you need to manual build whl and pip 
+it in local. 
+
+1. First, download Analytics Zoo automl source code from [GitHub](https://github.com/intel-analytics/analytics-zoo/tree/automl):
+    ```bash
+    git clone -b automl https://github.com/intel-analytics/analytics-zoo.git
```
Comment (jason-dai) [inline apps/automl/README.md]: no need to clone the code; they can simply download the code,

File: pyzoo/zoo/automl/README.md [hunk 2]
```diff
@@ -7,10 +7,32 @@ _A distributed **Automated Machine Learning** libary based on **ray** and **tens
 
 This library provides a framework and implementations for automatic feature engineering, model selection and hyper parameter optimization. It also provides a built-in automatically optimized model: _**TimeSequencePredictor**_ , which can be used for time series data analysis or anomaly detection. 
 
-Analytics Zoo AutoML is still in experimental stage and `pip install` haven't been supported yet. To use AutoML, you can download code of Analytics-zoo [automl branch](https://github.com/intel-analytics/analytics-zo/tree/automl). Then build the whl package manually following the instructions [here](https://analytics-zoo.github.io/master/#DeveloperGuide/python/#build-whl-package-for-pip-install) and intall the whl with pip in local.
-
-
-## 1 Automated Time Series Prediction 
+## 1 Install Analytics-zoo AutoML
```
Comment (jason-dai) [inline pyzoo/zoo/automl/README.md]: don't duplicate the document; you may refer to this guide in the readme of apps folder.

File: pyzoo/setup.py [hunk 3]
```diff
@@ -100,7 +100,10 @@ def setup_package():
         url='https://github.com/intel-analytics/analytics-zoo',
         packages=packages,
         install_requires=['pyspark==2.4.3', 'bigdl==0.10.0', 'conda-pack==0.3.1'],
-        extras_require={'ray': ['ray>=0.6.6', 'psutil', 'aiohttp', 'setproctitle']},
+        extras_require={'ray': ['ray>=0.6.6', 'psutil', 'aiohttp', 'setproctitle'],
+                        'automl': ['tensorflow>=1.14.0,<2.0.0', 'ray==0.7.0', 'psutil', 'aiohttp',
+                                   'setproctitle', 'pandas', 'featuretools', 'scikit-learn',
+                                   'numpy', 'requests']},
```
Comment (hkvision) [inline pyzoo/setup.py]: numpy is already included in bigdl, and thus no need here.

Comment (yushan111) [reply pyzoo/setup.py]: numpy has been removed

File: apps/automl/README.md (hunk 1 above)
Comment (yushan111) [reply apps/automl/README.md]: I have removed `git clone` part.

File: pyzoo/zoo/automl/README.md (hunk 2 above)
Comment (yushan111) [reply pyzoo/zoo/automl/README.md]: done

**Pull request merged**